zulipterminal/ui.py

```diff
@@ -77,6 +77,26 @@ def set_footer_text(self, text_list: Optional[List[Any]]=None,
             time.sleep(duration)
             self.set_footer_text()
 
+    @asynch
+    def set_typeahead_footer(self, suggestions: List[Any],
+                             state: Optional[int],
+                             is_truncated: bool) -> None:
+        # wrap by space and use only name from tuple
+        suggestions = list(map(lambda x: ' ' + x[1] + ' ', suggestions))
```

Comment:
You only use the second element of the tuple? Why are you passing in both?

See another comment, where I suggest unzipping the list of tuples first.

Reply:
Fixed by refactoring `match_stream`